src/patternfly/components/Table/examples/Table.md

```diff
@@ -654,12 +654,14 @@ When header cells are empty or they contain interactive elements, `<th>` should
 
 ## Expandable
 
+Note: Table contents will respond automatically to toggling table content. To retain column width, [column header and/or data cell behavior must be managed](#expandable-with-set-width-columns-example).
```

Comment:
I wonder if this might be better worded as something like this. Just more specifically calling out the column widths as changing and that you're assigning fixed widths. wdyt?

> Table column widths will respond automatically when toggling expanded rows. To retain column widths between expanded and collapsed states, column header and/or data cell widths must be set.